### ISSUE TITLE
setValue on this field not updating list or selected tags

### ISSUE THREAD

**Issue by victorsolaya** [markcunninghamuk/PCF_Library#1]
Hi!

I have checked that if you do a setValue("NAME") this is not appearing in the list or in the tags.
Would it be possible if you add this functionality to your PCF?

Thanks in advance!

**Comment by markcunninghamuk** [comment]
Yes we can. You can do a pull request or would you like me to do it?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by victorsolaya** [comment]
Hi Mark,
Up to you, I am more than happy to assist with anything you want 😄

**Comment by markcunninghamuk** [comment]
Let's try is the control working ok otherwise? Maybe show we a screenshot 😀

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by victorsolaya** [comment]
The control is quite good.
However, I've investigated in your code a little and I've realized you didn't write anything of control isDisabled, so that means that if you set the property as setDisabled(true) you still can select values in your PCF.
Another one that could be annoying is the fact every time you select a value it autocloses the list,
Apart from that, it works nicely. Probably I will do a PR at some point with some changes to "upgrade" a little bit your code.

Hopefully, this makes sense to you. It is a really good variant from the one I created 😀
